docs/vpm-spec.md

```diff
@@ -0,0 +1,428 @@
+The VPM Client then downloads the package archives from the URL provided by the Repository and extracts the package archive into `Packages` folder in the Unity project.
+The VPM Client also writes the managing package information to `Packages/vpm-manifest.json` to manage the installed packages.
+The VPM Client can re-install the package by using the information.
+This information is usually used to install packages after cloning the project with Version Control System.
+VPM Clients can share the user-wide installed repositories list and local package cache for better user experience.
+
+## 2. Conformance
+### 2.1. Syntax Notation
+This specification uses the Augmented Backus-Naur Form (ABNF) notation of [[RFC5234]].
+
+The following core rules are included by reference, as defined in Appendix B.1 of [[RFC5234]]: 
+ALPHA (letters), CR (carriage return), CRLF (CR LF), CTL (controls), DIGIT (decimal 0-9), DQUOTE (double quote), 
+HEXDIG (hexadecimal 0-9/A-F/a-f), HTAB (horizontal tab), LF (line feed), OCTET (any 8-bit sequence of data), SP (space), 
+and VCHAR (any visible US-ASCII character).
+
+### 2.2. Requirements Notation
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", 
+"**SHOULD NOT**", "**RECOMMENDED**", "**NOT RECOMMENDED**", "**MAY**", and "**OPTIONAL**" 
+in this document are to be interpreted as described in BCP 14 [[RFC2119]] [[RFC8174]] when, 
+and only when, they appear in all capitals, as shown here.
+
+## 3. Terminology and Core Semantics
+### 3.1. VPM, VPM Client, VPM Repository, VPM Profile Folder, and VPM Package
+The term "VPM" refers to the VRChat Package Manager, which is the package manager system.
+The VPM consists of the VPM Client and the VPM Repository.
+
+The VPM Client is the software that manages the VPM Packages in the Unity project.
+The VPM Client fetches the package information from the VPM Repository and installs the package.
+The VPM Client acts as a User Agent in the HTTP request to the VPM Repository.
+
+The VPM Clients usually have feature to manage VPM Repository installed for the VPM Manifest folder.
+The VPM Client **MAY** have feature to add VPM Repository from Web Browsers by using the `vcc:` URL scheme.
+
+The VPM Repository is the server that provides the package information and the package archive file.
+The VPM Repository provides the package information in [[JSON]] format and the package archive file in Unity Package format.
+One single VPM Repository can be consists of multiple servers, such as the package information server and the package archive server.
+The VPM Repository acts as an HTTP server to provide the package information and the package archive file.
+
+As described above, the VPM Client and VPM Repository communicate with each other by using the HTTP protocol [[HTTP]].
```

Comment:
nit: as above I expect HTTP to be well-known by readers of this document, can probably shortern to "As described above, the VPM Client and VPM Repository communicate with each other via HTTP."